pyproject.toml

```diff
@@ -0,0 +1,13 @@
+[tool.black]
```

Comment:
I moved this file to the root of the repo and added configuration to specify which files we expect to be formatted.  This'll be a bit of maintenance pain for the time being but at least codifies the demarcation.

Reply:
```
bblanken@wmbbe-67c seqr-loading-pipelines % black .
All done! ✨ 🍰 ✨
10 files left unchanged.
bblanken@wmbbe-67c seqr-loading-pipelines % echo $?
0
bblanken@wmbbe-67c seqr-loading-pipelines % isort .
Skipped 22 files
bblanken@wmbbe-67c seqr-loading-pipelines % echo $?
0
```